src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/InputFullOfBlockCommentsAst.txt

```diff
@@ -5,206 +5,198 @@ BLOCK_COMMENT_BEGIN -> /* [1:5]
-|   |   |   |   |   |   |--COMMENT_CONTENT -> 4 [1:34]
-|   |   |   |   |   |   `--BLOCK_COMMENT_END -> */ [1:34]
-|   |   |   |   |   `--IDENT -> puppycrawl [1:37]
-|   |   |   |   |--BLOCK_COMMENT_BEGIN -> /* [1:53]
-|   |   |   |   |   |--COMMENT_CONTENT -> 6 [1:55]
-|   |   |   |   |   `--BLOCK_COMMENT_END -> */ [1:55]
-|   |   |   |   `--IDENT -> tools [1:58]
-|   |   |   |--BLOCK_COMMENT_BEGIN -> /* [1:69]
-|   |   |   |   |--COMMENT_CONTENT -> 8 [1:71]
-|   |   |   |   `--BLOCK_COMMENT_END -> */ [1:71]
-|   |   |   `--IDENT -> checkstyle [1:74]
-|   |   `--IDENT -> grammars [1:85]
-|   |--BLOCK_COMMENT_BEGIN -> /* [1:99]
-|   |   |--COMMENT_CONTENT -> 10 [1:101]
-|   |   `--BLOCK_COMMENT_END -> */ [1:102]
-|   `--IDENT -> comments [1:105]
-|--BLOCK_COMMENT_BEGIN -> /* [1:113]
-|   |--COMMENT_CONTENT -> 11 [1:115]
-|   `--BLOCK_COMMENT_END -> */ [1:116]
-`--SEMI -> ; [1:119]
+|--ANNOTATIONS -> ANNOTATIONS [1:84]
+|--DOT -> . [1:84]
+|   |--BLOCK_COMMENT_BEGIN -> /* [1:63]
+|   |   |--COMMENT_CONTENT -> 7 [1:65]
+|   |   `--BLOCK_COMMENT_END -> */ [1:65]
+|   |--DOT -> . [1:68]
+|   |   |--BLOCK_COMMENT_BEGIN -> /* [1:47]
+|   |   |   |--COMMENT_CONTENT -> 5 [1:49]
+|   |   |   `--BLOCK_COMMENT_END -> */ [1:49]
+|   |   |--DOT -> . [1:52]
+|   |   |   |--BLOCK_COMMENT_BEGIN -> /* [1:26]
+|   |   |   |   |--COMMENT_CONTENT -> 3 [1:28]
+|   |   |   |   `--BLOCK_COMMENT_END -> */ [1:28]
+|   |   |   |--DOT -> . [1:31]
+|   |   |   |   |--BLOCK_COMMENT_BEGIN -> /* [1:17]
+|   |   |   |   |   |--COMMENT_CONTENT -> 2 [1:19]
+|   |   |   |   |   `--BLOCK_COMMENT_END -> */ [1:19]
+|   |   |   |   |--IDENT -> com [1:23]
+|   |   |   |   |--BLOCK_COMMENT_BEGIN -> /* [1:32]
+|   |   |   |   |   |--COMMENT_CONTENT -> 4 [1:34]
+|   |   |   |   |   `--BLOCK_COMMENT_END -> */ [1:34]
+|   |   |   |   `--IDENT -> puppycrawl [1:37]
+|   |   |   |--BLOCK_COMMENT_BEGIN -> /* [1:53]
+|   |   |   |   |--COMMENT_CONTENT -> 6 [1:55]
+|   |   |   |   `--BLOCK_COMMENT_END -> */ [1:55]
+|   |   |   `--IDENT -> tools [1:58]
+|   |   |--BLOCK_COMMENT_BEGIN -> /* [1:69]
+|   |   |   |--COMMENT_CONTENT -> 8 [1:71]
+|   |   |   `--BLOCK_COMMENT_END -> */ [1:71]
+|   |   `--IDENT -> checkstyle [1:74]
+|   `--IDENT -> astprinter [1:85]
+|--BLOCK_COMMENT_BEGIN -> /* [1:95]
+|   |--COMMENT_CONTENT -> 9 [1:97]
+|   `--BLOCK_COMMENT_END -> */ [1:97]
+`--SEMI -> ; [1:100]
 CLASS_DEF -> CLASS_DEF [3:6]
 |--MODIFIERS -> MODIFIERS [3:6]
-|   |--BLOCK_COMMENT_BEGIN -> /* [1:120]
-|   |   |--COMMENT_CONTENT -> 12 [1:122]
-|   |   `--BLOCK_COMMENT_END -> */ [1:123]
+|   |--BLOCK_COMMENT_BEGIN -> /* [1:101]
+|   |   |--COMMENT_CONTENT -> 10 [1:103]
+|   |   `--BLOCK_COMMENT_END -> */ [1:104]
 |   |--BLOCK_COMMENT_BEGIN -> /* [2:0]
-|   |   |--COMMENT_CONTENT -> 13 [2:2]
+|   |   |--COMMENT_CONTENT -> 11 [2:2]
 |   |   `--BLOCK_COMMENT_END -> */ [2:3]
 |   |--BLOCK_COMMENT_BEGIN -> /* [3:0]
-|   |   |--COMMENT_CONTENT -> 14 [3:2]
+|   |   |--COMMENT_CONTENT -> 12 [3:2]
```

Comment:
Why did comment text change so much in this file? All you changed was package name on first line, right?